DEVELOPMENT.md

```diff
@@ -40,25 +40,25 @@ This starts a CockroachDB and Redis database in Docker containers, creates a dat
 You can use the following commands to start, stop and erase databases.
 
 ```bash
-$ make dev.databases.start # Starts all databases in a Docker container
-$ make dev.databases.stop  # Stops all databases
+$ ./mage dev:dbStart # Starts all databases in a Docker container
+$ ./mage dev:dbStop  # Stops all databases
 
 # The contents of the databases will be saved in .dev/data.
 
-$ make dev.databases.erase # Stop all databases and erase storage.
+$ ./mage dev:dbErase # Stop all databases and erase storage.
```

Comment:
Let's make the comment consistent with the ones above while we're making changes.

> Stops all databases and erases storage.

And add periods to the comments above.